src/templates/best-cards/clans/ahrimanes.html

```diff
@@ -1,10 +1,10 @@
 {% extends "best-cards/_layout.html" %}
-{% import "best-cards/_layout_card_column.html" as layout with context%}
+{% import "best-cards/_layout_card_column.html" as layout with context %}
```

Comment:
Thanks for that ! Do you use a specific tool for Jinja template formatting ? Cause I'm searching for one for Visual Code.

Reply:
I'm using Atom.io but I editted it manually when I was looking for examples of your templating